nuxeo-features/preview/nuxeo-preview-core/src/test/java/org/nuxeo/ecm/platform/preview/tests/adapter/TestPreviewAdapter.java

```diff
@@ -152,4 +156,18 @@ public void testFolderishDocument() throws Exception {
         assertNotNull(adapter);
     }
 
+    @Test
+    public void testConvertTextToHtml() throws IOException {
+        Blob blob = new StringBlob("thisismytest", "text/plain");
+        DocumentModel doc = session.createDocumentModel("File");
```

Comment:
we can use the constructor with single parameter (which use `text/plain` mime type internally): 

`Blob blob = new StringBlob("thisismytest", "text/plain");` ->  `Blob blob = new StringBlob("thisismytest");`

Reply:
We could but I want to be explicit about the MIME type here given that it's what the test is about.